From 0c94f28051c5ec30d2881f253cd97beb0f0a4b2d Mon Sep 17 00:00:00 2001
From: Tom Anderson <thomasanderson@chromium.org>
Date: Mon, 22 Jul 2019 23:44:39 +0000
Subject: [PATCH] Fix Linux VAAPI build

Fixes the following error when generating build files:

ERROR Unresolved dependencies.
//media/gpu/vaapi:jpeg_decoder_unit_test(//build/toolchain/linux/unbundle:default)
  needs //media/capture:chromeos_test_utils(//build/toolchain/linux/unbundle:default)

BUG=None
R=mcasas

Change-Id: I2e4237c6b1e26bc97c26d6e5e0b22506d7a4758b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1713435
Auto-Submit: Thomas Anderson <thomasanderson@chromium.org>
Commit-Queue: Miguel Casas <mcasas@chromium.org>
Reviewed-by: Miguel Casas <mcasas@chromium.org>
Cr-Commit-Position: refs/heads/master@{#679768}
---

diff --git a/media/gpu/vaapi/BUILD.gn b/media/gpu/vaapi/BUILD.gn
index 5ec14eb..8481f9d 100644
--- a/media/gpu/vaapi/BUILD.gn
+++ b/media/gpu/vaapi/BUILD.gn
@@ -167,7 +167,6 @@
     ":vaapi_utils_unittest",
     "//base",
     "//media:test_support",
-    "//media/capture:chromeos_test_utils",
     "//media/parsers",
     "//testing/gtest",
     "//third_party/libyuv:libyuv",
@@ -175,6 +174,9 @@
     "//ui/gfx/codec",
     "//ui/gfx/geometry",
   ]
+  if (is_chromeos) {
+    deps += [ "//media/capture:chromeos_test_utils" ]
+  }
 }
 
 source_set("vaapi_utils_unittest") {
